sample-notificator-domain/src/main/groovy/life/qbic/business/notification/send/SendNotificationDataSource.groovy

```diff
@@ -9,4 +14,11 @@ package life.qbic.business.notification.send
  */
 interface SendNotificationDataSource {
 
+    /**
+     * Retrieves a list of subscribers, that need to be notified.
+     * Based on the current date (today) a list of subscribers that subscribed to an updated project will be returned.
+     * @param today The date of today
+     * @return a list of subscribers that need to be notified
+     */
+    List<Subscriber> getSubscribersForTodaysNotifications(Instant today) throws DatabaseQueryException
```

Comment:
As far as I understood it, you need to handle it project-based and not subscriber based. Thus this method would need to specify the project you want to retrieve the subscribers for.

Reply:
Really? AFAIK we will send notifications per person, so a notification should contain the collection of projects AND sample codes that were updated. So I think it makes sense to retrieve the subscribers that will be notified